static/src/stylesheets/global.scss

```diff
@@ -10,7 +10,6 @@
 @import 'module/_discussion-mixins';
 @import '_extends-only';
 
-
```

Comment:
looks like this is a whitespace only change? I think you want to revert this file!